### ISSUE TITLE
Is it possible to have a custom(non-date) placeholder?

### ISSUE THREAD

**Issue by bryceremick** [nathanreyes/v-calendar#347]
Is it possible to have a custom (non-date) placeholder on the text box? So for example, instead of it showing a date range or MM/DD/YYYY-MM/DD/YYYY, it would show custom text such as "Departure Window". 

A user would then hover over it, the calendar would pop up, and once a date is selected it would show the selected date in the text box.

Sorry if this is a dumb question, I just can't anything in the documentation about custom text-based placeholder.

**Comment by mofopeojosh** [comment]
Yes you can. You can use custom slot and use a custom input element. 'Departure window' can then be a placeholder on the input element.

**Comment by AntonioCholony** [comment]
```
<v-date-picker
 v-model="selectedDate"
 :input-props="{ class: &quot;date-picker&quot;, placeholder: &quot;Departure window'&quot;}"
/>
```
